<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>adminDetailPopup</class>
 <widget class="QWidget" name="adminDetailPopup">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>450</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>500</width>
    <height>450</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>500</width>
    <height>450</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Staff Details</string>
  </property>
  <widget class="QFrame" name="container_frame">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>480</width>
     <height>430</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
     QFrame#container_frame {
       background-color: white;
       border: 2px solid #3855DB;
       border-radius: 20px;
     }
    </string>
   </property>
   <widget class="QFrame" name="title_bar">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>480</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
      QFrame#title_bar {
        background-color: #3855DB;
        border-top-left-radius: 18px;
        border-top-right-radius: 18px;
      }
     </string>
    </property>
    <widget class="QLabel" name="label_title">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>10</y>
       <width>440</width>
       <height>30</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: white; font-size: 18px; font-weight: bold;</string>
     </property>
     <property name="text">
      <string>Admin Name</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_close">
     <property name="geometry">
      <rect>
       <x>440</x>
       <y>10</y>
       <width>30</width>
       <height>30</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">
       QPushButton { background-color: transparent; color: white; font-size: 20px; border: none; }
       QPushButton:hover { background-color: rgba(255,255,255,0.2); border-radius: 15px; }
      </string>
     </property>
     <property name="text">
      <string>x</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="content_widget" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>461</width>
      <height>361</height>
     </rect>
    </property>
    <widget class="QLabel" name="label_personal_info">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>150</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font-family: Fredoka-SemiBold; font-weight: bold;</string>
     </property>
     <property name="text">
      <string>Personal Info</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_email">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>35</y>
       <width>50</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Email:</string>
     </property>
    </widget>
    <widget class="QLabel" name="value_email">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>35</y>
       <width>250</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label_phone">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>50</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Phone:</string>
     </property>
    </widget>
    <widget class="QLabel" name="value_phone">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>60</y>
       <width>250</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_dateap">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>85</y>
       <width>100</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Date Applied:</string>
     </property>
    </widget>
    <widget class="QLabel" name="value_member_since">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>85</y>
       <width>200</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_order_stats">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>120</y>
       <width>150</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font-family: Fredoka-SemiBold; font-weight: bold;</string>
     </property>
     <property name="text">
      <string>Performance Metrics</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_order_process">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>160</y>
       <width>111</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Customer Made:</string>
     </property>
    </widget>
    <widget class="QLabel" name="value_order_process">
     <property name="geometry">
      <rect>
       <x>139</x>
       <y>160</y>
       <width>101</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_deliv_made">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>185</y>
       <width>100</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Staff Made:</string>
     </property>
    </widget>
    <widget class="QLabel" name="value_deliv_made">
     <property name="geometry">
      <rect>
       <x>139</x>
       <y>185</y>
       <width>101</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="value_last_order">
     <property name="geometry">
      <rect>
       <x>139</x>
       <y>210</y>
       <width>231</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="../images/Resc_rc.qrc"/>
 </resources>
 <connections/>
</ui>
